### ISSUE TITLE
FR: Better address scenario in logging where no current installed version of software is detected

### ISSUE THREAD

**Issue by apizz** [Installomator/Installomator#340]
Currently when debugging or just reviewing normal run output you get something like the text below when the software that's being installed isn't found to be installed already:

`2021-11-26 01:34:58 textexpander Downloaded version of TextExpander is 7.0.2 (replacing version ).`

The problem is it's not clear if the lack of a version number in `(replacing version )` is because of a bug or legitimately nothing found.  It would be helpful to simply remove this text if we're doing a new install (since other log messages indicate this fact) and only display the `(replacing version X)` message when this is an update of something that already exists.

**Comment by acodega** [comment]
I thought someone had a PR to address this already actually?

**Comment by Theile** [comment]
I think it was implemented in version 8.0.

**Comment by apizz** [comment]
It may have been implemented but it doesn't appear to be working as expected.  Here's some output from my system for software I don't have installed:

```
zsh assemble.sh -l ~/Documents/git/Installomator/fragments/labels textexpander
2022-01-05 15:34:00 textexpander ################## Start Installomator v. 9.0dev
2022-01-05 15:34:00 textexpander ################## textexpander
2022-01-05 15:34:00 textexpander DEBUG mode 1 enabled.
2022-01-05 15:34:01 textexpander BLOCKING_PROCESS_ACTION=tell_user
2022-01-05 15:34:01 textexpander NOTIFY=success
2022-01-05 15:34:01 textexpander LOGO=/System/Applications/App Store.app/Contents/Resources/AppIcon.icns
2022-01-05 15:34:01 textexpander no blocking processes defined, using TextExpander as default
2022-01-05 15:34:01 textexpander Changing directory to /Users/installomator/Documents/git/Installomator/build
2022-01-05 15:34:01 textexpander App(s) found: 
2022-01-05 15:34:01 textexpander could not find TextExpander.app
2022-01-05 15:34:01 textexpander appversion: 
2022-01-05 15:34:01 textexpander Latest version of TextExpander is 7.1
2022-01-05 15:34:01 textexpander TextExpander.dmg exists and DEBUG mode 1 enabled, skipping download
2022-01-05 15:34:01 textexpander DEBUG mode 1, not checking for blocking processes
2022-01-05 15:34:01 textexpander Installing TextExpander
2022-01-05 15:34:01 textexpander Mounting /Users/installomator/Documents/git/Installomator/build/TextExpander.dmg
2022-01-05 15:34:01 textexpander Mounted: /Volumes/TextExpander
2022-01-05 15:34:01 textexpander Verifying: /Volumes/TextExpander/TextExpander.app
2022-01-05 15:34:02 textexpander Team ID matching: 7PKJ6G4DXL (expected: 7PKJ6G4DXL )
2022-01-05 15:34:02 textexpander Downloaded version of TextExpander is 7.1 (replacing version ).
2022-01-05 15:34:02 textexpander DEBUG mode 1 enabled, skipping remove, copy and chown steps
2022-01-05 15:34:02 textexpander Finishing...
2022-01-05 15:34:12 textexpander App(s) found: 
2022-01-05 15:34:12 textexpander could not find TextExpander.app
2022-01-05 15:34:12 textexpander Installed TextExpander
2022-01-05 15:34:12 textexpander notifying
2022-01-05 15:34:13 textexpander Unmounting /Volumes/TextExpander
"disk4" ejected.
2022-01-05 15:34:13 textexpander DEBUG mode 1, not reopening anything
2022-01-05 15:34:13 textexpander ################## End Installomator, exit code 0 
```
